packages/zosfiles/src/cli/copy/ds/Ds.definition.ts

```diff
@@ -0,0 +1,69 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import { ICommandDefinition, ICommandOptionDefinition } from "@zowe/imperative";
+import { join } from "path";
+
+import i18nTypings from "../../-strings-/en";
+
+// Does not use the import in anticipation of some internationalization work to be done later.
+const strings = (require("../../-strings-/en").default as typeof i18nTypings).COPY.ACTIONS.DATA_SET;
+
+/**
+ * This object defines the command for copy data-set within zosfiles. This is not
+ * something that is intended to be used outside of the zosfiles package.
+ *
+ * @type {ICommandDefinition}
+ */
+export const DsDefinition: ICommandDefinition = {
+    name: "data-set",
+    aliases: ["ds"],
+    description: strings.DESCRIPTION,
+    type: "command",
+    handler: join(__dirname, "Ds.handler"),
+    profile: {
+        optional: ["zosmf"],
+    },
+    positionals: [
+        {
+            name: "fromDataSetName",
+            type: "string",
+            description: strings.POSITIONALS.FROMDSNAME,
+            required: true,
+        },
+        {
+            name: "toDataSetName",
+            type: "string",
+            description: strings.POSITIONALS.TODSNAME,
+            required: true,
+        },
+    ],
+    options: ([
+    ] as ICommandOptionDefinition[]).sort((a, b) => a.name.localeCompare(b.name)),
```

Comment:
Should we expose all ICopyDataSetOptions?
https://github.com/zowe/zowe-cli/blob/master/packages/zosfiles/src/api/methods/copy/doc/ICopyDatasetOptions.ts

Reply:
I was planning on delivering the core functionality to solve #580 and then start working on #581 to add support for all the options. This is because the list of available options is still growing as #579 is still being worked on.
However, if you think it'd be best to include enqueue and replace support in this PR, I can close this PR temporarily